src/test/java/unit/tests/utils/MyDateTests.java

```diff
@@ -0,0 +1,334 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package unit.tests.utils;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import utils.MyDate;
+
+import java.util.Calendar;
+
+public class MyDateTests {
+
+    @Test(dataProvider = "validInputData")
+    public void validateInputshouldReturnTrue(String input) {
+        MyDate myDate = new MyDate();
+        Assert.assertTrue(myDate.validateInput(input));
+    }
+
+    @Test(dataProvider = "invalidInputData")
+    public void validateInputshouldReturnFalse(String input) {
+        MyDate myDate = new MyDate();
+        Assert.assertFalse(myDate.validateInput(input));
+    }
+
+    @DataProvider(name = "validInputData")
+    public static Object[][] validInputData() {
+        return new Object[][]{
+                {"3 days ago"},
+                {"10 months ago"},
+                {"3 years from now"},
+                {"500 seconds ago"},
+                {"1000 hours from now"}
+        };
+    }
+
+    @DataProvider(name = "invalidInputData")
+    public static Object[][] invalidInputData() {
+        return new Object[][]{
+                {"0 months from now"},
+                {"1 hours from now"},
+                {"2 hour ago"}
+        };
+    }
+
+    @Test
+    public void getTodayShouldReturnTheCorrectDate() throws Exception {
+        MyDate myDate = new MyDate();
+        Calendar c;

```

Comment:
Change your variable name "c" for another name that is more descriptive

Reply:
Ty will be fixed.